src/features/collections/data_products/HeatMap.module.scss

```diff
@@ -2,10 +2,76 @@
 
 $label-len: 20em;
 
+:global(div.dragcover) {
```

Comment:
any way to get more specific here? I worry it may affect other plots in the future

Reply:
Unfortunately, this is plotly behavior which is not very configurable. As we discussed, the ways to improve this situation are complicated and best saved for a circumstance where a problem is encountered.